Code/Editor/Util/GuidUtil.cpp

```diff
@@ -6,12 +6,35 @@
  *
  */
 
-
-#include "EditorDefs.h"
-
 #include "GuidUtil.h"
 
+const char* GuidUtil::ToString(REFGUID guid)
+{
+    static char guidString[64];
```

Comment:
If this is static then this function won't be thread safe

Reply:
yes, this is thread unsafe and its being used from UI thread only, this was in the header, I just moved it to the cpp